libopenage/input/input_manager.cpp

```diff
@@ -30,13 +30,13 @@ std::string mod_set_string(modset_t mod) {
 			switch (it) {
 			case modifier::ALT:
```

Comment:
Indent here?

Reply:
also thought like this, but mom.cpp says something different

Reply:
Oh sorry, I totally mixed this up with `libopenage/options.cpp`. The indentations there must go I think.

Reply:
done